README.md

```diff
@@ -1,41 +1,66 @@
-## [Download TripleA](http://triplea-game.org/download/)
+# TripleA: Open Source Strategy Game Engine
 
-TripleA (2002) is a fan created open source strategy and board game engine where you can play Axis and
-Allies style rules on community created maps.TripleA has a well-developed AI and an active lobby for live
-multiplayer games.
+![Game Board Screenshot](https://user-images.githubusercontent.com/12397753/36015523-a4e28a24-0d23-11e8-84c0-c4bd0ee19ce0.png)
 
-In TripleA, players can recreate World War 2 with the Axis pushing through to Moscow and the Allies
-desperately fighting back, Napoleon's march across Europe, Rome defeating the Carthaginian Empire,
-or even Sauron conquering Middle Earth!
+## About TripleA
 
+TripleA is a fan-created, open-source strategy and board game engine that brings Axis and Allies-style gameplay to life. Launched in 2002, TripleA offers:
 
-  - ***[Forums: Discussion, Questions & Help](https://forums.triplea-game.org/category/10/help-questions)***
-  - ***[How to Contribute to TripleA](/docs/contribute.md)***
-  - ***[Bug Tracker](https://github.com/triplea-game/triplea/issues/new)***
-  - ***[Developer Setup Guide](/docs/development/README.md)***
-  - ***[![TripleA license](https://img.shields.io/github/license/triplea-game/triplea.svg?style=flat-square)](https://github.com/triplea-game/triplea/blob/master/LICENSE)***
+- Community-created maps
+- Well-developed AI for single-player experiences
+- Active lobby for live multiplayer games
+- Diverse scenarios ranging from World War II to fantasy realms
 
-![Game Board Screenshot](https://user-images.githubusercontent.com/12397753/36015523-a4e28a24-0d23-11e8-84c0-c4bd0ee19ce0.png)
+## Key Features
+
+- **Historical Scenarios**: Recreate pivotal moments in history, such as:
+  - The Axis push towards Moscow in WWII
+  - Napoleon's march across Europe
+  - Rome's conquest of the Carthaginian Empire
+- **Fantasy Realms**: Dive into imaginary worlds, like Sauron's conquest of Middle Earth
+- **Customizable Gameplay**: Enjoy a wide variety of community-created maps and scenarios
+
+![Europe Scenario](https://user-images.githubusercontent.com/12397753/132109225-71e6c02d-425e-4b8d-9537-7ac66baebbfd.jpeg)
+
+![Middle Earth Scenario](https://user-images.githubusercontent.com/12397753/132109223-14a0aa2e-a950-4a5e-9937-3c4b52211cd9.jpeg)
+
+## Getting Started
+
+### Download
+Get the latest version of TripleA from our official website:
```

Comment:
I felt that the download link being up front and top was important. I do like the new flow to introduce the project first and then the "getting started"